### PR TITLE
Add express server

### PR DESCRIPTION
**Pull request by MrGRA**
Add express server
add some endpoints

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const port = 3000
+
+app.get('/', (req, res) => {
+  res.json({version:'V1 API'})
+})
+
+app.get('/user', (req,res) => {
+  console.log(req.params)
+  res.json({
+    username: 'MrGRA',
```
Comment (marteaup) [inline app/index.js:12]: should it be hard coded?

File: init.js [hunk 2]
```diff
@@ -0,0 +1 @@
+require('./app')
```
Comment (marteaup) [inline init.js]: missing line! please add an empty line at the bottom

**Review by marteaup — CHANGES_REQUESTED** [review]
please answer the questions and do the modifications.

File: app/index.js (hunk 1 above)
Comment (MrGRA) [reply app/index.js:12]: Yes, this will be changed afterward.

**Review by marteaup — APPROVED** [review]

**Pull request merged**